package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@vueuse/core": "^7.5.4",
+    "floating-vue": "^2.0.0-beta.6",
```

Comment:
Is this necessary? I can't seem to find where it's used besides global registration.

Reply:
oh whoops that was something i was playing with on my own, i'll take that out of the pr